projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -185,7 +186,7 @@ jQuery( document ).ready( function ( $ ) {
 			var $activate_button = $( this );
 
 			// Do not allow any requests if the button is disabled.
-			if ( $activate_button.attr( 'disabled' ) ) {
+			if ( $activate_button.attr( 'disabled' ) || $activate_button.attr( 'link' ) ) {
```

Comment:
Curious here, wouldn't this disable clicks on the updated settings button?

Reply:
Yeah was trying to test this bit and whether it sent me to the link, clicking settings without it started to try and activate the module again.